semconvgen/generator.go

```diff
@@ -48,6 +49,7 @@ func main() {
 	flag.StringVarP(&cfg.outputFilename, "filename", "f", "", "Filename for templated output. If not specified 'basename(inputPath).go' will be used.")
 	flag.StringVarP(&cfg.templateFilename, "template", "t", "template.j2", "Template filename")
 	flag.StringVarP(&cfg.templateParameters, "parameters", "p", "", "List of key=value pairs separated by comma. These values are fed into the template as-is.")
+	flag.StringVarP(&cfg.capitalizationsPath, "capitalizations-path", "z", "", "Path to a file of newline separated capitalization strings.")
```

Comment:
@dmathieu should we add "`that will be appended to the default list`" here as well?

Reply:
Cool, I can add this and change the README to match it.